Skip test when /dev/log unavailable.  Avoids a problem on buildds where syslogd is not running.

diff -Naur --exclude=debian perl-5.10.0.orig/ext/Sys/Syslog/t/syslog.t perl-5.10.0/ext/Sys/Syslog/t/syslog.t
--- perl-5.10.0.orig/ext/Sys/Syslog/t/syslog.t	2007-12-18 21:47:07.000000000 +1100
+++ perl-5.10.0/ext/Sys/Syslog/t/syslog.t	2007-12-30 12:54:30.379668340 +1100
@@ -189,6 +189,9 @@
     skip "the 'unix' mechanism works, so the tests will likely fail with the 'stream' mechanism", 10 
         if grep {/unix/} @passed;
 
+    skip "can't connect to Unix socket: _PATH_LOG unavailable", 10
+      unless -e Sys::Syslog::_PATH_LOG();
+
     # setlogsock() with "stream" and an undef path
     $r = eval { setlogsock("stream", undef ) } || '';
     is( $@, '', "setlogsock() called, with 'stream' and an undef path" );
